### PR TITLE
feat: testnet label

### PR DESCRIPTION
**Pull request by 0xtiti**

### CODE REVIEW

File: src/containers/Header.tsx [hunk 1]
```diff
@@ -63,6 +63,7 @@ export const Header = () => {
             )}
             {isMobile && <Image src={responsiveLogo} alt='Superchain Bridge' priority />}
           </Link>
+          <StyledLabel className='testnet-label'>testnet</StyledLabel>
```
Comment (0xArdy) [inline src/containers/Header.tsx:66]: it should appear only in the testnet ui

File: src/containers/Header.tsx [hunk 2]
```diff
@@ -196,3 +197,23 @@ const StyledSettingsIcon = styled(Image)({
...
+    borderRadius: '0.75rem',
+    border: `1px solid ${currentTheme.ghost[400]}`,
+    background: currentTheme.ghost[400],
+    left: '1rem',
+    padding: '0rem 0.5rem',
+    alignItems: 'center',
+    boxShadow: '0px 1px 2px 0px rgba(16, 24, 40, 0.05)',
+    color: currentTheme.steel[950],
+    '@media (max-width: 600px)': {
+      top: '-0.1rem',
```
Comment (0xArdy) [inline src/containers/Header.tsx:216]: it should be vertically aligned on mobile
image

**Review by 0xArdy — CHANGES_REQUESTED** [review]

File: src/containers/Header.tsx (hunk 2 above)
Comment (0xtiti) [reply src/containers/Header.tsx]: [a1a72f0](https://github.com/defi-wonderland/superchain-bridge-ui/pull/77/commits/a1a72f0c4815149c46e30365a32eba4953f3afcc)

File: src/containers/Header.tsx (hunk 1 above)
Comment (0xtiti) [reply src/containers/Header.tsx]: sorry about missing this! 👀   [a1a72f0](https://github.com/defi-wonderland/superchain-bridge-ui/pull/77/commits/a1a72f0c4815149c46e30365a32eba4953f3afcc)

**Review by 0xArdy — APPROVED** [review]
lgtm!

**Review by agusduha — APPROVED** [review]

**Pull request merged**